app/views/layouts/_user_menu.html.haml

```diff
@@ -16,6 +16,11 @@
     = link_to(conference_program_tracks_path(@conference.short_title)) do
       %span.fa.fa-road
       My Tracks
+-if @conference && @conference.program && (@conference.program.cfps.for_booths.try(:open?) || current_user.booths.where(conference_id: @conference.id).count > 0)
```

Comment:
@nasia-sam by the way, instead of `current_user.booths.where(conference_id: @conference.id).count > 0` you can simply do `current_user.booths.any?`